clients/onboarding/src/pages/company/ownership-beneficiary/OnboardingCompanyOwnershipBeneficiaryFormCommon.tsx

```diff
@@ -89,6 +87,8 @@ export const OnboardingCompanyOwnershipBeneficiaryFormCommon = forwardRef<
     .with("ESP", "FRA", "NLD", "ITA", () => true)
     .otherwise(() => false);
 
+  const [initialBirthdate] = useState(Option.fromNullable(initialValues.birthDate).toUndefined());
```

Comment:
why do you use a `state` here? also the conversion can be `initialValues.birthDate ?? undefined`